utils/common/src/test/java/org/apache/brooklyn/util/javalang/MemoryUsageTrackerTest.java

```diff
@@ -82,7 +82,7 @@ private long sizeOfActiveReferences(List<Maybe<byte[]>> references) {
         return size;
     }
 
-    @Test(groups="Integration")
+    @Test(groups="Live")
```

Comment:
See https://github.com/apache/brooklyn-server/pull/426. Let's see if that fixes it. If you want to disable the test in the meantime, that's fine with me as well.